changelog/fragments/1669049916-CHANGELOG.asciidoc.yaml

```diff
@@ -0,0 +1,31 @@
+# Kind can be one of:
+# - breaking-change: a change to previously-documented behavior
+# - deprecation: functionality that is being removed in a later release
+# - bug-fix: fixes a problem in a previous version
+# - enhancement: extends functionality but does not break or fix existing behavior
+# - feature: new functionality
+# - known-issue: problems that we are aware of in a given version
+# - security: impacts on the security of a product or a user’s deployment.
+# - upgrade: important information for someone upgrading from a prior version
+# - other: does not fit into any of the other categories
+kind: feature
+
+# Change summary; a 80ish characters long description of the change.
+summary: Adds support for a new beat for container drift prevention.
```

Comment:
The changes here are enough to let the agent run your new input when it is manually packaged with the agent, but it won't be usable as part of any release until you get it hooked into to the agent release process. 

We may want to hold off on adding release notes until that is done, but if you plan to hook into the release process for 8.7 we can just leave this here.

Reply:
cc @lrishi

Reply:
Thanks, created a work item: https://github.com/elastic/anduril/issues/35